docs/deploy-ibm.md

```diff
@@ -0,0 +1,246 @@
+
+:::
+
+![K8S Creation Image 11](/assets/IBMCloud/k8s11.png)
+
+3. After running the second and third commands shown in the above images, you will get an output as below. You do not need to run the first command because you've already logged in to IBM Cloud.
+
+Output
+![Output of Command IKSConnect](/assets/IBMCloud/IKSConnect.png)
+
+> Now we are able to run commands of kubectl.
+
+## 4. Verify the deployment
+Make sure all the pods of Pulsar are running. Get the service URL and broker URL for publishing and consuming the messages.
+
+
+
+> [Installation of Pulsar helm in minikube cluster](https://pulsar.apache.org/docs/getting-started-helm/)  this document is for minikube cluster we are taking reference from that.
+
+1. Add Pulsar chart repo
+
+```bash
+$ helm repo add apache https://pulsar.apache.org/charts
+$ helm repo update
+$ helm repo list
+```
+
+Output
+![Output of Command Helmrepo](/assets/IBMCloud/Helmrepo.png)
+
+2. Clone the Pulsar Helm chart repository, go inside the pulsar-helm-chart directory
+
+```bash
+git clone https://github.com/apache/pulsar-helm-chart   
+cd pulsar-helm-chart
+```
+
+3. Run the script`prepare_helm_release.sh`to create secrets required for installing the Apache Pulsar Helm chart. The username`pulsar`and password`pulsar`are used for logging into the Grafana dashboard and Pulsar Manager.
+
+:::note
+
+When running the script, you can use`-n`to specify the Kubernetes namespace where the Pulsar Helm chart is installed,`-k`to define the Pulsar Helm release name, and`-c`to create the Kubernetes namespace. For more information about the script, run `./scripts/pulsar/prepare_helm_release.sh --help`.
+
+:::
+
+```bash
+ ./scripts/pulsar/prepare_helm_release.sh -n default -k asia -c 
+```
+
+Output
+![Output of Command HelmOutPut](/assets/IBMCloud/HelmOutPut.png)
+
+4. Use the Pulsar Helm chart to install a Pulsar cluster to Kubernetes.
+
+```bash
+helm install --values examples/values-minikube.yaml --set initialize=true asia apache/pulsar
+```
+
+:::note
+
+You need to specify`--set initialize=true`when installing Pulsar the first time. This command installs and starts Apache Pulsar.
+
+:::
+
+Output
+![Output of Command HelmInstall](/assets/IBMCloud/HelmInstall.png)
+
+5. Check the status of all pods.
+
+```bash 
+kubectl get pods
+```
+If all pods start up successfully, you can see that `STATUS`is changed to`Running`or`Completed`.
+
+Output
+
+![Output of Command PodStatus](/assets/IBMCloud/PodStatus.png)
+
+6. Check the status of all services.
+
+```bash
+ kubectl get services
+``` 
+
+Output
+
+![Output of Command Services](/assets/IBMCloud/Services.png)
+
+The output shows both `services URL` and `broker URL`. 
+Proxy external IPs are the ports changed just now:
+- Service URL port is 80.
+- Broker URL port is 6650. This is default.
+If you execute all the commands successfully, you can use the Pulsar client to connect to ?? clusters and produce and consume messages through proxy external IPs.
```

Comment:
Is this an unfinished paragraph with "?? clusters"?

Reply:
ping @5exceptions

Reply:
Thanks @momo-jun , @Anonymitaet 
We have fixed all the reported issue regarding the formatting. Please review the latest commit.